tests/installation/install_and_reboot.pm

```diff
@@ -170,13 +170,11 @@ sub run {
         select_console 'installation';
     }
 
-    prepare_system_shutdown;
```

Comment:
This call does some magic for s390x as well, so not sure if we should remove it

Reply:
I wonder about this remove myself. The idea by mgriessmeier and me was to abstract the arch and machine specific stuff but that *is* what mnowak is intending with power_action with the observe flag. I think it's exactly as we want it but we should test this PR on the different variants we have. @Soulofdestiny could you help verify on the different s390x variants? @nsinger could help with the ipmi backend verification

Reply:
@rwx788 Well, `prepare_system_shutdown` is removed only technically as it's run soon in `power_action`.
@okurz Correct.

The take-away message here is that whenever we restart (or let the system restart) w/o going thru the pains of `power_action`, there's a goof chance it won't work on svirt - especially on Xen - as expected. There are probably several such occurrences in our code, I'll deal with then on case by case basis.

Reply:
Thanks for clarification!